consul/state/tombstone_gc_test.go

```diff
@@ -102,3 +102,18 @@ func TestTombstoneGC_Expire(t *testing.T) {
 	case <-time.After(20 * time.Millisecond):
 	}
 }
+
+func TestTombstoneGC_TimerAfterShutdown(t *testing.T) {
+	ttl := 2 * time.Microsecond
+	gran := 1 * time.Microsecond
+	gc, err := NewTombstoneGC(ttl, gran)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	for i := 0; i < 1000; i++ {
```

Comment:
This would repro the panic, but probably isn't useful as an ongoing unit test since it's not super deterministic (I had to push it up to 1000 to get it to fail). I'll remove it.